tests/enums/test_generic.py

```diff
@@ -0,0 +1,131 @@
+from unittest import TestCase
+
+from vstools import ChromaLocation, FieldBased, Resolution, UnsupportedFieldBasedError, vs
+
+
+class TestChromaLocation(TestCase):
+    def test_from_res_uhd(self) -> None:
+        clip = vs.core.std.BlankClip(format=vs.YUV420P8, width=3840, height=2160)
+        result = ChromaLocation.from_res(clip)
+        self.assertEqual(result, ChromaLocation.LEFT)
```

Comment:
Is this result correct? Shouldn't this be CENTER?

Reply:
Good question. But currently, it is returning LEFT for this, as all the tests are passing.

Reply:
It looks like `ChromaLocation.from_res` branches on the colorimetry, not the resolution. It will return TOP_LEFT if the primaries, matrix, or transfer are BT2020.

Reply:
Yeah, we checked and decided it's best to just return LEFT always. See https://github.com/Jaded-Encoding-Thaumaturgy/vs-tools/commit/8db06593900915f6628224c09dba9243b8b510d0.